client/modules/workspace/src/constants.ts

```diff
@@ -17,3 +17,28 @@ export const STATUS_TEXT_MAP: Record<string, string> = {
 };
 
 export const TERMINAL_STATES = [`FINISHED`, `CANCELLED`, `FAILED`];
+
+export const QUEUE_LIST: Record<string, Record<string, string>> = {
```

Comment:
I think we should make these changes on the tapis system rather than in the codebase. In the Tapis System queue definition, there are separate `name` and `hpcQueueName` fields such that we can distinguish between the slurm queue name and a label we can use in the UI

Reply:
Discussed offline - we'll try implementing a `queueFilter` on the system definition itself per portal, for a portal-wide system queue filter